src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -1419,7 +1419,7 @@ public bool Ellipsis
         /// The ellipsis position type when the text size over the layout size.<br />
```

Comment:
```
TextEditor
EllipsisPosition { get; set; } // Property
Ellipsis { get; set; }
```

You also need to open the TextEditor's Ellipsis Property.

Reply:
@wonrst , done